include/idf/EthernetExtreme3dPro.hh

```diff
@@ -0,0 +1,36 @@
+/*
+PURPOSE:
+LIBRARY DEPENDENCIES: (
+(idf/EthernetExtreme3dPro.cpp)
+)
+*/
+
+/**
+ * @trick_parse{everything}
+ * @trick_link_dependency{idf/EthernetExtreme3dPro.cpp}
+ */
+
+#ifndef ETHERNET_EXTREME_3D_PRO_HH
+#define ETHERNET_EXTREME_3D_PRO_HH
+
+#include "idf/EthernetWingMan.hh"
+#include "idf/Extreme3dPro.hh"
+
+#include <string>
+
+namespace idf {
+
+class EthernetExtreme3dPro : public EthernetWingMan, public Extreme3dPro {
```

Comment:
just to close this out since we talked offline: The Extreme 3D Pro is from the Wingman line of Logitech devices. the data sent is one additional byte for the extra buttons. The other bytes all map 1:1